plugin/documents.py

```diff
@@ -97,14 +97,6 @@ def notify() -> None:
 
         sublime.set_timeout_async(notify)
 
-    def on_reload_async(self) -> None:
-        for listener in list(self.view_listeners):
-            listener.on_reload_async()
-
-    def on_revert_async(self) -> None:
-        for listener in list(self.view_listeners):
-            listener.on_revert_async()
```

Comment:
The alternative would be to keep calling here and instead rename the methods in `DocumentSyncListener`. I think those two options would have slightly different behavior but I'm not sure what's the most correct way.

One thing that `TextChangeListener` does that `DocumentSyncListener` doesn't is it checks if the view is a `is_regular_view`.